lib/createTestFromScenario.js

```diff
@@ -141,6 +146,9 @@ const createTestFromScenarios = (
 
           const scenarioExample = {
             ...section,
+            // tags on scenario's should be inherited by examples (https://cucumber.io/docs/cucumber/api/#tags)
+            tags: example.tags.concat(section.tags),
+            shouldRun: shouldRunExamples,
```

Comment:
@Sonaryr @MateuszNowosad @lgandecki This line introduced the regression in #612. This `shouldRun` does not consider `@focus`.

`shouldRun` is previously set [here](https://github.com/TheBrainFamily/cypress-cucumber-preprocessor/blob/master/lib/createTestsFromFeature.js#L31), but this overwrites it.